app/dashboard/views.py

```diff
@@ -5344,3 +5347,92 @@ def bulkDM(request):
     }
 
     return TemplateResponse(request, 'bulk_DM.html', context)
+
+
+def validate_number(user_id, twilio, phone, redis):
+    validation = twilio.lookups.phone_numbers(phone).fetch(type=['caller-name', 'carrier'])
+
+    if validation.caller_name and validation.caller_name["caller_type"] == 'CONSUMER':
```

Comment:
might be useful to have the whole `validation` object stored in the DB as a json object so we can look at it down the line.

we should of course hash the phone number bc its PII when we do so tho.